main.gms

```diff
@@ -1762,6 +1762,9 @@ $setglobal cm_INCONV_PENALTY_FESwitch  on !! def = on  !! regexp = off|on
 *** *JH/LB* Activate MOFEX partial fossil fuel extraction cost minimization model
 *** * Warning: Use a well-converged run since the model uses vm_prodPe from the input GDX
 $setGlobal cm_MOFEX  off        !! def = off  !! regexp = off|on
+*** cm_limitSolidsFossilRegi off   !! def=off
+*** ext_regi value, e.g. "EUR_regi, USA"   "solids fossil in industry and buildings for regions within EUR_regi and USA are forced to be monotonuosly decreasing from 2020 or cm_startyear onward."
```

Comment:
I personally wouldn't be 100% sure how to interpret the new line starting with "ext_regi value" - maybe it would be clearer to start the line with "replace 'off' by ext_regi value, e.g. " ?